Library/OcPciIoLib/OcPciIoLib.c

```diff
@@ -0,0 +1,76 @@
+/** @file
+   PciIo overrides
+
+   Copyright (c) 2023, xCuri0. All rights reserved.<BR>
+   SPDX-License-Identifier: BSD-2-Clause-Patent
+
+ **/
+
+#include "OcPciIoU.h"
+#include <Library/OcPciIoLib.h>
+
+EFI_CPU_IO2_PROTOCOL *
+OcPciIoInstallProtocol (
+  IN BOOLEAN  Reinstall
+  )
+{
+  EFI_TPL                          tpl;
+  UINTN                            Index;
+  EFI_STATUS                       Status  = 0;
+  EFI_CPU_IO2_PROTOCOL             *mCpuIo = NULL;
+  UINTN                            HandleCount;
+  EFI_HANDLE                       *HandleBuffer;
+  EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL  *PciRootBridgeIo;
+
+  if (Reinstall) {
+    return NULL;
```

Comment:
What’s the point of the parameter if it just turns the function into a no-op?

Reply:
Changed to return mCpuIo

Reply:
Why is this done before the check below?